skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg

```diff
@@ -1,4 +1,15 @@
 # PointOfInterestDetails.Text(Data)
-- Here are details.
+- ${PointOfInterestDetailsTextAndSpeak1(Data)}
 
+# PointOfInterestDetailsTextAndSpeak1(Data)
+[Activity
```

Comment:
This doesn't contain Text property. would it successfully generate an activity with Text property?

Reply:
or do you just want Text property to be empty?

Reply:
Yes, then the text is null in generated activity.
It follows the previous visual design.